### DISCUSSION TITLE
Comments in Markdown Page Prevent Rendering

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3851] [Error / Bug Report]
Hello,

I noticed after the last update that now you can't update and render a page while it has HTML comments  in it. 

It renders when you edit the page in markdown editor preview, but when you save the page, it doesn't render any of your changes and uses whatever the last save was.

**Comment by Kyushin** [comment]
Hello,
Same issue here.
I tried with  and with other way founded in the web but got same result.

**Comment by honzaplojhar** [comment]
Hi, 
I just found the same issue. 

When I insert comment `<!-- +text+ -->` in markdown (preview is displayed correctly without commented text) and save, last page version without comment is published instead. 

Is it a bug or do I use it in a wrong way? Thanks!

**Comment by rrrevin** [comment]
Just found this post. Re-rendering all pages under utilities doesn't fix this either.
This bug just cost me 4 days of frustration! P L E A S E fix :)

**Comment by FarmDoc** [comment]
Update: With 2.5.300,  HTML comments in markdown do not longer prevent rendering. 
They are (correctly) not shown in the Preview, but are shown as normal text in the page, once Edit mode is closed.

I used `[//]: # (This may be the most platform independent comment)` as  proposed here: https://stackoverflow.com/questions/4823468/comments-in-markdown

Still, HTML comments would be nice.
